plugins/modules/mso_backup.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2020, Shreyas Srish (@shrsr) <ssrish@cisco.com>
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+__metaclass__ = type
+
+ANSIBLE_METADATA = {'metadata_version': '1.1',
+                    'status': ['preview'],
+                    'supported_by': 'community'}
+
+DOCUMENTATION = r'''
+---
+module: mso_backup
+short_description: Manages backups
+description:
+- Manage backups on Cisco ACI Multi-Site.
+author:
+- Shreyas Srish (@shrsr)
+options:
+  location_type:
+    description:
+    - The type of location for the backup to be stored
+    type: str
+    choices: [ local, remote]
+    default: local
+  backup:
+    description:
+    - The name given to the backup
+    type: str
+    aliases: [ name ]
+  remote_location:
+    description:
+    - The remote location's name for the backup to be stored
+    type: str
+  remote_path:
+   description:
+    - This path is relative to the remote location.
+    - There's no need to specify '/' for this path.
```

Comment:
I would rephrase with "A '/' is automatically added between the remote location folder and this path."
I would also add 'This folder structure should already exist on the remote location'.